tools/integration_tests/util/operations/file_operations.go

```diff
@@ -16,47 +16,68 @@
 package operations
 
 import (
+	"bytes"
 	"crypto/rand"
 	"fmt"
 	"io"
+	"io/fs"
 	"log"
 	"os"
 	"os/exec"
+	"strconv"
+	"strings"
 	"syscall"
 )
 
-func CopyFile(srcFileName string, newFileName string) (err error) {
```

Comment:
This function may get use over many places. e.g. In [operations](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/operations/copy_file_test.go#L42). Please fix that in all the places, and test once all the tests in sync are working properly.

Reply:
> Please fix that in all the places

I haven't removed the function, only bifurcated it into two. Nor have I changed its signature in any way.

Before change: CopyFile(): Didn't allow overwriting destination file passed.
After change: 
1. CopyFile(): Still doesn't allow overwriting destination file passed. Defined right after the common business logic function `copyFile`.
2. CopyFileAllowOverwrite(): Does allow overwriting a pre-existing destination file passed.

Reply:
Also, I have tested by running all the integration tests in Louhi pipeline.